src/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -609,10 +609,16 @@ public static class MsalError
         /// <para>For more details</para> See https://aka.ms/msal-net-enable-keychain-groups
         /// </summary>
         public const string MissingEntitlements = "missing_entitlements";
+
+        /// <summary>
+        /// Xamarin.iOS + broker specific. This error indidates that the UiParent is null.
+        /// <para>Mitigation</para> In order to access the broker, you will need to pass in a UIViewController.
+        /// <para>For more details</para> see https://aka.ms/msal-net-ios-broker
+        /// </summary>
+        public const string UIViewControllerRequiredForiOSBroker = "uiviewcontroller_required_for_ios_broker";
```

Comment:
actually, this is because the uiviewcontroller is null. we should always get a uiparent w/iOS. i forgot to update the comment to reflect this. will do.

---
In reply to: [300964227](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1251#discussion_r300964227) [](ancestors = 300964227)

Reply:
@jean-Marc Prieur and I synced on this off-line.

---
In reply to: [301177937](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1251#discussion_r301177937) [](ancestors = 301177937,300964227)